product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py

```diff
@@ -43,19 +43,14 @@ def setUp(self):
     def _create_customer(self, name):
         """Create a Partner."""
         return self.env["res.partner"].create(
-            {
-                "name": name,
-                "email": "example@yourcompany.com",
-                "customer": True,
-                "phone": 123456,
-            }
+            {"name": name, "email": "example@yourcompany.com", "phone": 123456}
         )
 
     def _create_partnerinfo(
-        self, supplierinfo_type, partner, product, empty_variant=False
+        self, supplierinfo_type, customer, product, empty_variant=False
```

Comment:
why are we renaming this argument?

Reply:
partner_id is a mandatory many2one field in the sale.order model